package.json

```diff
@@ -1,54 +1,55 @@
 {
   "name": "dutch-tax-income-calculator2",
-  "version": "0.0.0",
+  "version": "1.1.0",
   "license": "MIT",
-  "angular-cli": {},
   "scripts": {
+    "ng": "ng",
     "start": "ng serve",
-    "lint": "tslint \"src/**/*.ts\"",
+    "build": "ng build --aot=false --base-href ./",
+    "build:prod": "ng build --prod --aot=false --base-href \"https://eduardomourar.github.io/dutch-tax-income-calculator2/\"",
```

Comment:
make sure you to review this line and change it if you are deploying to github page.